Client/Frontend/Browser/BrowserNavigationHelper.swift

```diff
@@ -52,10 +52,10 @@ class BrowserNavigationHelper {
         open(vc, doneButton: DoneButton(style: .done, position: .right))
     }
     
-    func openDownloads() {
-        guard let bvc = bvc else { return }
-        let vc = DownloadsPanel(profile: bvc.profile)
-        open(vc, doneButton: DoneButton(style: .done, position: .right))
+    func openDownloads(_ completion: @escaping (Bool) -> Void) {
+        FileManager.default.openBraveDownloadsFolder() {
+            completion($0)
+        }
```

Comment:
Fun thing that you absolutely do not have to do for this PR to merge: You can just pass completion as a parameter like: `FileManager.default.openBraveDownloadsFolder(completion)` instead of making a closure to call the closure.

Reply:
Fixed at https://github.com/brave/brave-ios/pull/4769/commits/45302f2174c558639506d226f45d4f45ec43b447